### ISSUE TITLE
[calendar] Moving an all-day event causes the event to end at 1am

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#419]
```
When moving an all-day event via drag&drop, it is turned into an event starting 
at 12am and ending at 1am.

What steps will reproduce the problem?
1. open calendar (month or week view)
2. create new all-day event (00:00-23:59)
3. drag&drop the event to any other day

What is the expected output?
The event remains an all-day event.

What do you see instead?
The event becomes a 1-hour event starting at 00:00 and ending at 01:00.

What version of the product are you using? On what operating system?
9.9.6.9.3

```

Original issue reported on code.google.com by `sten...@gmail.com` on 5 Nov 2012 at 12:48

**Comment by GoogleCodeExporter** [comment]
Original comment by `roland.l...@gmail.com` on 5 Nov 2012 at 2:02
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed now. Please use Plugin Manager Center to open tickets next 
time. Thanks!
```

Original comment by `roland.l...@gmail.com` on 5 Nov 2012 at 2:11
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
I confirm that the issue has been fixed w/ 9.9.6.9.4.

Sorry, I will use the Plugin Manager Center from now on. Thanks a lot!
```

Original comment by `sten...@gmail.com` on 5 Nov 2012 at 3:01
